python-sdk/src/transfers/utils.py

```diff
@@ -0,0 +1,24 @@
+from airflow.hooks.base import BaseHook
+from transfers.datasets.base import UniversalDataset as Dataset
+
+
+def check_if_connection_exists(conn_id: str) -> bool:
+    """
+    Given an Airflow connection ID, identify if it exists.
+    Return True if it does or raise an AirflowNotFoundException exception if it does not.
+
+    :param conn_id: Airflow connection ID
+    :return bool: If the connection exists, return True
+    """
+    try:
+        BaseHook.get_connection(conn_id)
```

Comment:
I wonder if there is a more friendly way of checking if a connection exists 🤔

Reply:
I was using airflow basehook like we do in astro-sdk